pages/linux/readpe.md

```diff
@@ -0,0 +1,28 @@
+# readpe
+
+> Displays information about PE files.
```

Comment:
Does PE stand for `Portable Executable`? If yes I'd say we should mention that:
```suggestion
> Displays information about PE (Portable Executable) files.
```

Reply:
Cfr [this page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/readelf.md) ; ELF is not specified.
I think the meaning of PE is straightforward for people using this tool...